backend/Squirrel.Core/Squirrel.Core.WebAPI/Controllers/StaticFilesController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Squirrel.Core.BLL.Interfaces;
+
+namespace Squirrel.Core.WebAPI.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class StaticFilesController : Controller
+{
+    private readonly IStaticFilesService _staticFilesService;
+
+    public StaticFilesController(IStaticFilesService staticFilesService)
+    {
+        _staticFilesService = staticFilesService;
+    }
+
+    [HttpGet("downloadSquirrelInstaller"), DisableRequestSizeLimit]
```

Comment:
we don't use camelCase and verbs in routes, so rename it to "squirrel-installer"